src/main/java/racingcar/InputView.java

```diff
@@ -1,15 +1,34 @@
 package racingcar;
 
+import java.util.Scanner;
+
 public class InputView {
 
+    private static final Scanner scanner = new Scanner(System.in);
```

Comment:
상수 컨벤션에 따라 이름은 대문자로 작성해보면 어떨까요?
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```

Reply:
깜빡했네요